tests/puppeteer/pages/BO/customers/addresses/index.js

```diff
@@ -8,22 +8,16 @@ module.exports = class Addresses extends BOBasePage {
     this.pageTitle = 'Addresses •';
 
     // Selectors
-    // Header links
-    this.addNewAddressLink = '#page-header-desc-configuration-add[title=\'Add new address\']';
     // List of addresses
     this.addressGridPanel = '#address_grid_panel';
     this.addressGridTitle = `${this.addressGridPanel} h3.card-header-title`;
-    this.addressesListForm = '#address_grid';
-    this.addressesListTableRow = `${this.addressesListForm} tbody tr:nth-child(%ROW)`;
-    this.addressesListTableColumn = `${this.addressesListTableRow} td.column-%COLUMN`;
-    this.addressesListTableColumnAction = this.addressesListTableColumn.replace('%COLUMN', 'actions');
-    this.addressesListTableToggleDropDown = `${this.addressesListTableColumnAction} a[data-toggle='dropdown']`;
-    this.addressesListTableDeleteLink = `${this.addressesListTableColumnAction} a[data-url]`;
-    this.addressesListTableEditLink = `${this.addressesListTableColumnAction} a[href*='edit']`;
+    this.addresssListForm = '#address_grid';
```

Comment:
There is a type in `this.addresssListForm`, shouldn't it be `this.addressesListForm`?